fcollada.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix=@DEST_DIR@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
+
+Name: fcollada
+Description: FCollada library for working with COLLADA 3D models
+Version: github.com/matthewlai/fcollada/commit/9325c758e76b218829cca82863a2389c9fc72534
```

Comment:
I missed this one just yet, please fix this line, although I don't really have a strong opinion on what it should be

Reply:
Ah sorry about that. Put that in for testing and forgot about it.

I've looked into it some more, and it looks like although there are no strict rules to follow, pkgconfig uses RPM rules for comparison, so dotted decimal is the most idiomatic (though I'm not sure if anyone actually uses pkg-config for dependencies, so it may not be terribly important).

How about 3.5.1.0? ( 3.5.1 = 3.05B, 0 is the package maintainer's version that can be incremented if anyone complains?)

Reply:
Fine with me.

Reply:
Done